server/repo.go

```diff
@@ -42,8 +42,7 @@ func PostRepo(c *gin.Context) {
 
 	repo.IsActive = true
 	repo.UserID = user.ID
-	if !repo.AllowPush && !repo.AllowPull && !repo.AllowDeploy && !repo.AllowTag {
-		repo.AllowPush = true
+	if !repo.AllowPull {
```

Comment:
This invert c0nf - can you ad a smal explanation

Reply:
I will remove it. It has been used before if all checks are disabled they assumed the user hasn't specified it in the update post and took some default values as deactivating all webhooks would disable the whole repo.